doc/classes/ProjectSettings.xml

```diff
@@ -108,12 +111,14 @@
 			<argument index="0" name="name" type="String">
 			</argument>
 			<description>
+			  	Returns the initial value of the specified property. Returns null if it does not exist.
```

Comment:
Returns null if *what* does not exist? The property? Or the initial value of the property?

Sorry, just being a little nit-picky.

Reply:
The property, will clarify